wormhole-connect/src/routes/types.ts

```diff
@@ -1,37 +0,0 @@
-import { TransferInfo } from '../utils/sdkv2';
```

Comment:
Should we delete this file since it's now empty?

Reply:
thought I had. thanks